### ISSUE TITLE
Some properties of StructureDefinition do not need Export.ExternalizeReferences 

### ISSUE THREAD

**Issue by whyfate** [FirelyTeam/spark#235]
current:
```xml
<StructureDefinition xmlns="http://hl7.org/fhir">
  <id value="Patient"/>
  .....
 <type value="https://localhost:5001/fhir/Patient"/>
</StructureDefinition>
```
hope:
```xml
<StructureDefinition xmlns="http://hl7.org/fhir">
  <id value="Patient"/>
  .....
 <type value="Patient"/>
</StructureDefinition>
```

**Comment by hryh** [comment]
When I read the StructureDefinition resource, the type property is added a prefix with "https://localhost:5001/fhir/", and property StructureDefinition.context.type too, and so on.

**Comment by kennethmyhra** [comment]
Thanks @hryh for reporting, this seems like a bug, but it probably affects all attributes of type Uri and has probably been like this for a long time. I'll label it as a bug that needs further research.

**Comment by kennethmyhra** [comment]
Fixed in version 1.5
